### ISSUE TITLE
File moving should not be done on a patch version

### ISSUE THREAD

**Issue by soullivaneuh** [ebidel/filer.js#61]
Apparently, you moved a file to the `dist` folder on patch version: https://github.com/ebidel/filer.js/compare/0.4.4...0.4.5

This is completely BC break. For my case, my asset compiler broke because of a not found file:

``` bash
  [RuntimeException]                                                                       
  The source file "/code/app/../web/assets/lib/filer.js/src/filer.min.js" does not exist.  
```

If you follow [semver](http://semver.org/) conventions, this kind of things should be done on a major version for a stable project or a minor for an unstable `0.x` one (this is your case AFAIK), but never on a patch version.

This is OK now and I hope this issue will help people searching why the new patch version is not working anymore.

Thank you fore reading and considering this issue. :wink: 

Regards

**Comment by ebidel** [comment]
Yes you're right this should have been a minor version and explained in the release notes.
